tools/realsense-viewer/realsense-viewer.cpp

```diff
@@ -501,7 +501,12 @@ int main(int argc, const char** argv) try
             }
         }
 
-        ImGui::SetNextWindowSize({ viewer_model.panel_width, 20.f * (new_devices_count + multiline_devices_names) + 8 + (is_ip_device_connected ? 0 : 20) });
+        float line_h = ImGui::GetTextLineHeightWithSpacing() + 2;
```

Comment:
Please explain this changes

Reply:
`float line_h = ImGui::GetTextLineHeightWithSpacing() + 2;`

- This function returns the height of a line of text, taking into account any additional spacing between lines as defined by the ImGui style. The returned value includes both the height of the text itself and any extra vertical spacing that might be present.
- I add 2 because we have some extra pixels above and under line.

`float separator_h = new_devices_count > 1 ? ImGui::GetStyle().ItemSpacing.y : 0;`

- `ImGui::GetStyle().ItemSpacing.y` gives us a height of spacing (Separator line)
- I using ternary construction because we add a separator only when devices_count > 1 (see line 557)

Reply:
Thanks